plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/__init__.py

```diff
@@ -71,12 +71,26 @@ class RayNodeConf(Dict[str, Any]):
     ImageId: str = MISSING
 
 
+@dataclass
+class RayCodeSyncConf:
+    """
+    If your code contains more than one modules/files, you need to sync your code to the remote cluster.
+    The code will be synced to a tmp dir on the cluster.
+    code_dir could be a relative dir to where you run your app or a absolute dir.
+    Leave this empty if you don't need to sync code.
+    """
+
+    code_dir: str = ""
+    include: List[str] = field(default_factory=list)
+    exclude: List[str] = field(default_factory=list)
```

Comment:
document these.
Are they being passed rsync? to which flags?

Reply:
good. remove the TODO. I don't want that lingering there. if we need to support more things we will add explicit support to specific arguments.